BootloaderCommonPkg/Include/Guid/OsBootOptionGuid.h

```diff
@@ -71,8 +71,8 @@ typedef enum {
 typedef enum  {
   EnumFileSystemTypeFat,
   EnumFileSystemTypeExt2,
-  EnumFileSystemTypeAuto,
-  EnumFileSystemMax
+  EnumFileSystemMax,
+  EnumFileSystemTypeAuto = (1 < EnumFileSystemTypeFat) | (1< EnumFileSystemTypeExt2)
```

Comment:
I think it might be be better to use the bits instead to make it straightforward.
typedef enum  {
  EnumFileSystemTypeFat   = BIT0,
  EnumFileSystemTypeExt2 = BIT1,
  EnumFileSystemTypeAll    = BIT0 | BIT1,
  EnumFileSystemMax	    = EnumFileSystemTypeAll + 1
}

Max should be the last item.

Reply:
Sorry,  I meant the style needs to be consistent.  If sequential number is used, then bit mask should not be used at the same time.    You can also use something like below:
ypedef enum {
EnumFileSystemTypeFat,
EnumFileSystemTypeExt2,
EnumFileSystemTypeAuto,
EnumFileSystemMax
}

Reply:
Thanks for the comments, will update the code